emote/callbacks/logging.py

```diff
@@ -53,7 +53,7 @@ def end_cycle(self, bp_step, bp_samples):
 
             for k, v in cb.windowed_scalar_cumulative.items():
                 k_split = k.split("/")
-                k_split[0] = k_split[0] + "_" + suffix
+                k_split[0] = f"cumulative/{k_split[0]}_{suffix}"
                 k = "/".join(k_split)
 
                 self._writer.add_scalar(f"{k}/cumulative", v, bp_step)
```

Comment:
We could do "cumulative/{k}" when adding the scalar instead to avoid the duplication

Reply:
I made it redundant so it would be easy to exclude when you search for it. But I am happy to skip it.